api/test/test_edr.py

```diff
@@ -10,6 +10,98 @@
 client = TestClient(app)
 
 
+def test_get_locations_without_query_params():
+    with patch("routers.edr.get_obs_request") as mock_get_obs_request, patch(
+        "routers.edr.verify_parameter_names"
+    ) as mock_verify_parameter_names:
+        test_data = load_json("test/test_data/test_feature_collection_proto.json")
+        compare_data = load_json("test/test_data/test_feature_collection.json")
```

Comment:
This test is really about testing that the mock gets called with the correct parameters, correct?

It is probably good to have a comments around `test_data` that is is just "random" data, not appropriate for the request. And I would also drop the `compare_data` and the `assert response.json() == compare_data` for the same reason.

That part is tested later anyway.